lib/Alchemy/Phrasea/Controller/Api/V1Controller.php

```diff
@@ -869,10 +869,12 @@ public function substituteAction(Request $request)
                 'You do not have access to collection %s', $collection->get_label($this->app['locale.I18n'])
             ));
         }
-        $record->substitute_subdef($request->get('name'), $media, $this->app);
+        $adapt = $request->get('adapt')===null || !(\p4field::isno($request->get('adapt')));
```

Comment:
Second part of the expression is not used. Assignment has precedence over boolean comparison. Not sure about the value you wish to have is `$adapt` variable